### ISSUE TITLE
Add a simple object detection example 

### ISSUE THREAD

**Issue by marianopeck** [vasmalltalk/tensorflow-vast#5]
Now that we got image recognition to work, we shoud try "Object Detection". Something like [this](http://andrew.carterlunn.co.uk/programming/2018/01/24/annotating-large-datasets-with-tensorflow-object-detection-api.html) or [this](https://blog.algorithmia.com/deep-dive-into-object-detection-with-open-images-using-tensorflow/) or [this](https://www.kdnuggets.com/2018/03/google-tensorflow-object-detection-api-the-easiest-way-implement-image-recognition.html).

I wonder if we can download pre-trained models with that and then display the images from Smalltalk. Did you ever try? @SergeStinckwich ?

**Comment by marianopeck** [comment]
There are some opinions already about this:

https://github.com/vasmalltalk/tensorflow-vast/issues/4#issuecomment-518448398

https://github.com/vasmalltalk/tensorflow-vast/issues/4#issuecomment-518484149

https://github.com/vasmalltalk/tensorflow-vast/issues/4#issuecomment-518485168

**Comment by gerasdf** [comment]
I've been looking at the "TensorFlow Object Detection Zoo" (https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md) and it's absolutely possible (and somehow easy) to implement all of them...

I just need to have some time. The basic structure is the same as for `LabelImage` examples, so we may want to add a common superclass that loads the graph and handles a number of image files. Then what's different is the input node (we can search it by name, it's easy) and the output tensors and labels file format... We'll see... I'm swamped with work, so I can't really sit down and do it, but maybe... just maybe...

**Comment by SergeStinckwich** [comment]
If we split the effort we can do most of the example in the webpage I guess.
